Sources/main.cpp

```diff
@@ -154,6 +269,21 @@ int main(int argc, char **argv) try {
             case 'c':
                 certPath = optarg;
                 break;
+            case 'f':
+                if (strcmp(optarg, "-") == 0) {
+                    GetArchiveFileListFromMappingFile(std::cin, fileNames);
+                } else {
+                    std::ifstream file;
+                    file.exceptions(std::ifstream::failbit);
```

Comment:
I had that, but it wasn't working on macOS. I found the issue with that though. I will upload a revised version.